Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -158,4 +173,13 @@ namespace ROS2
     {
         required.push_back(AZ_CRC("TransformService"));
     }
+
+    ROS2FrameComponent::ROS2FrameComponent()
+    {
+    }
```

Comment:
FYI you can default this constructor here
```suggestion
    ROS2FrameComponent::ROS2FrameComponent() = default;
```